### PR TITLE
Fix formatting and enable hack/man-page-checker

### PR DESCRIPTION
**Pull request by rwha**
This allows the `hack/man-page-checker` script to run successfully. I had to make some uninformed decisions regarding the descriptive text differences. Any input feedback would be appreciated!

Signed-off-by: Ryan Whalen <rj.whalen@gmail.com>

### CODE REVIEW

**Comment by rwha** [comment]
I see `podman-remote` is a bit of an outlier. I'll see if I can find a good solution

**Comment by rhatdan** [comment]
LGTM. 
My only issue is if I do a 
man podman-container-list it is going to give me information on podman-ps, with no message about the command I executed.

**Comment by rhatdan** [comment]
@edsantiago @TomSweeneyRedHat PTAL

**Comment by rhatdan** [comment]
Is this script run on each PR?  We probably should block PRs based on this script passing.

**Comment by rwha** [comment]
There's a comment at the top of the script:

> FIXME as of 2019-03-20 there are still four files with inconsistent names:
>
>    podman-logs.1.md         NAME= podman-container-logs
>    podman-info.1.md         NAME= podman-system-info
>    podman-rm.1.md           NAME= podman-container-rm
>    podman-rmi.1.md          NAME= podman-image-rm
>
> If those four get renamed (with suitable symlink fixes), this script
> can be enabled in CI to prevent future inconsistencies.

those four are fixed with this PR, I believe, however some others needed some attention.

File: docs/links/podman-image-rm.1 [hunk 1]
```diff
@@ -1 +1 @@
-.so man1/podman-rm.1
+.so man1/podman-rmi.1
```
Comment (TomSweeneyRedHat) [inline docs/links/podman-image-rm.1]: I don't think we should do this change?  Is there a reason why you think it's needed?

Comment (rhatdan) [reply docs/links/podman-image-rm.1]: That looks correct to me.
`podman-image-rm` is the same as `podman rmi`
`podman-container-rm is the same as `podman rm`

Comment (rwha) [reply docs/links/podman-image-rm.1]: `man podman-image-rm` gives the manpage for podman-rm (podman-container-rm)

File: docs/podman-container.1.md [hunk 2]
```diff
@@ -14,8 +14,8 @@ The container command allows you to manage containers
 | Command    | Man Page                                            | Description                                                                  |
 | ---------  | --------------------------------------------------- | ---------------------------------------------------------------------------- |
 | attach     | [podman-attach(1)](podman-attach.1.md)              | Attach to a running container.                                               |
-| checkpoint | [podman-container-checkpoint(1)](podman-container-checkpoint.1.md)  | Checkpoints one or more containers.                          |
-| cleanup    | [podman-container-cleanup(1)](podman-container-cleanup.1.md)    | Cleanup containers network and mountpoints.                      |
+| checkpoint | [podman-container-checkpoint(1)](podman-container-checkpoint.1.md)  | Checkpoints one or more running containers.                  |
+| cleanup    | [podman-container-cleanup(1)](podman-container-cleanup.1.md)    | Cleanup container network and mountpoints.                       |
```
Comment (TomSweeneyRedHat) [inline docs/podman-container.1.md]: suggest:
s/container network/the container's network/
Also you've an extra space at the end of the line that will be problematic for the CI checks.

File: docs/podman-image.1.md [hunk 3]
```diff
@@ -25,10 +25,10 @@ The image command allows you to manage images
...
 | rm       | [podman-rmi(1)](podman-rmi.1.md)                | Removes one or more locally stored images.                                  |
 | save     | [podman-save(1)](podman-save.1.md)              | Save an image to docker-archive or oci.                                     |
-| sign     | [podman-image-sign(1)](podman-image-sign.1.md)  | Sign an image.                                                              |
+| sign     | [podman-image-sign(1)](podman-image-sign.1.md)  | Create a signature for an image                                             |
 | tag      | [podman-tag(1)](podman-tag.1.md)                | Add an additional name to a local image.                                    |
 | tree     | [podman-image-tree(1)](podman-image-tree.1.md)  | Prints layer hierarchy of an image in a tree format.                        |
-| trust    | [podman-image-trust(1)](podman-image-trust.1.md)| Manage container image trust policy.                                        |
+| trust    | [podman-image-trust(1)](podman-image-trust.1.md)| Manage container registry image trust policy                                |
```
Comment (TomSweeneyRedHat) [inline docs/podman-image.1.md]: missing period at end for these two changes.

File: docs/podman-logs.1.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
 % podman-logs(1)
 
 ## NAME
-podman\-container\-logs (podman\-logs) - Fetch the logs of one or more containers
+podman\-logs - Display the logs of a container
```
Comment (TomSweeneyRedHat) [inline docs/podman-logs.1.md]: I'd suggest  s/one or more containers/a container/

File: docs/podman-play.1.md [hunk 5]
```diff
@@ -14,7 +14,7 @@ file input.  Containers will be automatically started.
 
 | Command  | Man Page                                            | Description                                                                  |
 | -------  | --------------------------------------------------- | ---------------------------------------------------------------------------- |
-| kube     | [podman-play-kube(1)](podman-play-kube.1.md)        | Recreate pods and containers based on Kubernetes YAML.                       |
+| kube     | [podman-play-kube(1)](podman-play-kube.1.md)        | Create pods and containers based on Kubernetes YAML.                            |
```
Comment (TomSweeneyRedHat) [inline docs/podman-play.1.md]: Think you've extra space at the end of this line too.

File: docs/podman-pod-kill.1.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 % podman-pod-kill(1)
 
 ## NAME
-podman\-pod\-kill - Kills all containers in one or more pods with a signal
+podman\-pod\-kill - Kill the main process of each container in pod
```
Comment (TomSweeneyRedHat) [inline docs/podman-pod-kill.1.md]: suggest:
s/in one or more pods/in pod/

File: docs/podman-pod.1.md [hunk 7]
```diff
@@ -11,22 +11,22 @@ podman pod is a set of subcommands that manage pods, or groups of containers.
...
+| Command | Man Page                                          | Description                                                                    |
+| ------- | ------------------------------------------------- | --------------------------------------------------------------------------------- |
+| create  | [podman-pod-create(1)](podman-pod-create.1.md)    | Create a new pod.                                                                 |
+| exists  | [podman-pod-exists(1)](podman-pod-exists.1.md)    | Check if a pod exists in local storage.                                           |
+| inspect | [podman-pod-inspect(1)](podman-pod-inspect.1.md)  | Displays information describing a pod.                                            |
+| kill    | [podman-pod-kill(1)](podman-pod-kill.1.md)        | Kill the main process of each container in pod.                                   |
+| pause   | [podman-pod-pause(1)](podman-pod-pause.1.md)      | Pause one or more pods.                                                           |
+| prune   | [podman-pod-prune(1)](podman-pod-prune.1.md)      | Remove all stopped pods                                                           |
```
Comment (TomSweeneyRedHat) [inline docs/podman-pod.1.md]: missing period

File: docs/podman-remote.1.md [hunk 8]
```diff
@@ -1,7 +1,7 @@
 % podman-remote(1)
 
 ## NAME
-podman-remote - A remote CLI for Podman: A Simple management tool for pods, containers and images.
+podman-remote - A tool for remote management of pods, containers and images
```
Comment (TomSweeneyRedHat) [inline docs/podman-remote.1.md]: I'm not in favor of this change.  I'll let @baude and @jwhonce chime in.

File: docs/podman-unshare.1.md [hunk 9]
```diff
@@ -1,10 +1,10 @@
 % podman-unshare(1)
 
 ## NAME
-podman\-unshare - Run a command inside of a modified user namespace.
+podman\-unshare - Run a command inside of a modified user namespace
 
 ## SYNOPSIS
-**podman unshare** [*options*] [**--**] [*command*]
+**podman unshare** [*options*] [*\--*] [*command*]
```
Comment (TomSweeneyRedHat) [inline docs/podman-unshare.1.md]: is the slash right here?

File: hack/man-page-checker [hunk 10]
```diff
@@ -40,7 +40,7 @@ done
 #
 # Make sure the descriptive text in podman-foo.1.md matches the one
 # in the table in podman.1.md.
-for md in *.1.md;do
+for md in *-*.1.md;do
```
Comment (TomSweeneyRedHat) [inline hack/man-page-checker]: @edsantiago WDYT?

File: docs/links/podman-image-rm.1 (hunk 1 above)
Comment (TomSweeneyRedHat) [reply docs/links/podman-image-rm.1]: Thanks, my dyslexia must have been kicking in, never saw the image part of that.

File: docs/podman-container.1.md (hunk 2 above)
Comment (TomSweeneyRedHat) [reply docs/podman-container.1.md]: Oops, no extra space, you've an ending bar that just <cr>'d on me in the github review.

File: hack/man-page-checker (hunk 10 above)
Comment (rwha) [reply hack/man-page-checker]: This prevents podman.1.md itself from being included in the checks for parent descriptions

**Comment by rwha** [comment]
well I shouldn't have forced an amend, apologies.

File: docs/podman-remote.1.md (hunk 8 above)
Comment (rwha) [reply docs/podman-remote.1.md]: I'm trying to find a way to exclude podman-remote from the hack/man-page-checker description check; I'll revert this change.

Comment (rhatdan) [reply docs/podman-remote.1.md]: for md in $(ls -1 *-*.1.md | grep -v remote);do

**Comment by edsantiago** [comment]
Beautiful work, thank you so much!

/lgtm

File: docs/podman-system-renumber.1.md [hunk 11]
```diff
@@ -1,7 +1,7 @@
 % podman-system-renumber(1)
 
 ## NAME
-podman\-system\-renumber - Renumber container locks
+podman\-system\-renumber - Migrate lock numbers to handle a change in maximum number of locks
```
Comment (TomSweeneyRedHat) [inline docs/podman-system-renumber.1.md]: Migrate isn't a good word here, it brings up a bad connotation.  Suggest: "Acquire container locks to handle a change in the maximum number of locks"

**Pull request merged**